gen/docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.16
+
+RUN apt-get update \
```

Comment:
⚠️ **[hadolint]** <[DL3008](https://github.com/hadolint/hadolint/wiki/DL3008)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`